apps/analytics/urls.py

```diff
@@ -24,7 +24,7 @@
         name="get_challenge_phase_submission_analysis",
     ),
     url(
-        r"^challenge/(?P<challenge_pk>[0-9]+)/challenge_phase/(?P<challenge_phase_pk>[0-9]+)/count$",
+        r"^challenge/(?P<challenge_pk>[0-9]+)/challenge_phase/(?P<version>(v1|v2))/(?P<challenge_phase_pk_or_slug>[-a-zA-Z0-9_]+)/count$",
```

Comment:
Hey are we required to do this API also using slug in this PR?

Reply:
yes, as in `vm.getResults()` we are calling both submissions as well as count. So, both will be called using the slug only.